ci/cron/src/Main.hs

```diff
@@ -234,7 +237,7 @@ main = do
     else do
         Temp.withTempDir $ \docs_folder -> do
             putStrLn "Building docs listing"
-            build_docs_folder docs_folder $ gh_resp & List.sortBy compare_versions & reverse & map name
+            build_docs_folder docs_folder $ gh_resp & map name & List.sortOn to_v & reverse
```

Comment:
I find this unnecessarily hard to read because I have to scan from both sides. Could you please either use `$` or `&` but not both. I generally prefer `$` because it it gives you the same order as function application.
Also:
```suggestion
            build_docs_folder docs_folder $ gh_resp & map name & List.sortOn (Down . to_v)
```
where `Down` is from `Data.Ord`.

Reply:
Done.